javascript-carousel/main.js

```diff
@@ -0,0 +1,99 @@
+var $right = document.querySelector('.fa-angle-right');
+var $left = document.querySelector('.fa-angle-left');
+var $bulbasoarButton = document.querySelector('.bulbasoar');
+var $charmanderButton = document.querySelector('.charmander');
+var $squirtleButton = document.querySelector('.squirtle');
+var $pikachuButton = document.querySelector('.pikachu');
+var $jigglypuffButton = document.querySelector('.jigglypuff');
+var $img = document.querySelector('img');
+var $outerdiv = document.querySelector('.outerdiv');
+
+var view = 'bulbasoar';
+
+$outerdiv.addEventListener('click', function (event) {
+  if (event.target.tagName === 'BUTTON') {
+    if (event.target.className === 'bulbasoar') {
+      $img.setAttribute('src', 'images/001.png');
+      view = 'bulbasoar';
+    } else if (event.target.className === 'charmander') {
+      $img.setAttribute('src', 'images/004.png');
+      view = 'charmander';
+    } else if (event.target.className === 'squirtle') {
+      $img.setAttribute('src', 'images/007.png');
+      view = 'squirtle';
+    } else if (event.target.className === 'pikachu') {
+      $img.setAttribute('src', 'images/025.png');
+      view = 'pikachu';
+    } else if (event.target.className === 'jigglypuff') {
+      $img.setAttribute('src', 'images/039.png');
+      view = 'jigglypuff';
+    }
+    changeButton();
+  }
+});
+
+$right.addEventListener('click', nextImage);
+
+function nextImage() {
+  if (view === 'bulbasoar') {
+    $img.setAttribute('src', 'images/004.png');
+    view = 'charmander';
+  } else if (view === 'charmander') {
+    $img.setAttribute('src', 'images/007.png');
+    view = 'squirtle';
+  } else if (view === 'squirtle') {
+    $img.setAttribute('src', 'images/025.png');
+    view = 'pikachu';
+  } else if (view === 'pikachu') {
+    $img.setAttribute('src', 'images/039.png');
+    view = 'jigglypuff';
+  } else if (view === 'jigglypuff') {
+    $img.setAttribute('src', 'images/001.png');
+    view = 'bulbasoar';
+  }
```

Comment:
This is entirely built around the order of the images. If anybody ever wanted to reorder the images, everything would break colossally. You should make a function that can work no matter what order the images are in.